nototools/data/noto_cmap_phase3.xml

```diff
@@ -19,7 +19,7 @@
     <tr>Bamu,Bamum,661,0000 000d 0020 00a0 a6a0-a6f7 16800-16a38,0,</tr>
     <tr>Bass,Bassa Vah,45,0000 000d 0020 0022 002c 002e 00a0 201c-201d 16ad0-16aed 16af0-16af5,-1,</tr>
     <tr>Batk,Batak,74,0000 000d 0020 00a0 00d7 1bc0-1bf3 1bfc-1bff 200b-200d 2012-2015 2022 25cc 25fb-25fe,-1,</tr>
-    <tr>Beng,Bengali,157,0000 000d 0020-0023 0025 0027-003f 005b-005f 007b-007e 00a0 00ad 00d7 00f7 0951-0952 0964-0965 0980-0983 0985-098c 098f-0990 0993-09a8 09aa-09b0 09b2 09b6-09b9 09bc-09c4 09c7-09c8 09cb-09ce 09d7 09dc-09dd 09df-09e3 09e6-09fb 200b-200d 2010 2013-2014 2018-2019 201c-201d 2026 20b9 2212 25cc a8f1,-1,</tr>
+    <tr>Beng,Bengali,157,0000 000d 0020-0023 0025 0027-003f 005b-005f 007b-007e 00a0 00ad 00d7 00f7 02bc 0951-0952 0964-0965 0980-0983 0985-098c 098f-0990 0993-09a8 09aa-09b0 09b2 09b6-09b9 09bc-09c4 09c7-09c8 09cb-09ce 09d7 09dc-09dd 09df-09e3 09e6-09fb 200b-200d 2010 2013-2014 2018-2019 201c-201d 2026 20b9 2212 25cc,-1,</tr>
```

Comment:
Drop U+A8F1?  I thought we had decided to keep it in Bengali?  Oh. I saw the comment below in the code. Ok.